package.json

```diff
@@ -50,8 +50,8 @@
     "typescript": "^4.9.3"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.2.0 || ^10.0.0",
-    "@nestjs/core": "^9.2.0 || ^10.0.0",
+    "@nestjs/common": "^9.2.0 || ^10.0.0 || ^11.0.0",
+    "@nestjs/core": "^9.2.0 || ^10.0.0 || ^11.0.0",
     "reflect-metadata": "^0.1.13"
```

Comment:
@tumainimosha would be nice to also have `"reflect-metadata": "^0.1.13 || ^0.2.1"`, so we won't get warnings when installing the package (or errors in strict resolution mode)